technic/machines/other/constructor.lua

```diff
@@ -102,6 +103,9 @@ local function make_on(mark, length)
 			minetest.check_for_falling(pos)
 			for i = 1, length do
 				place_pos = vector.add(place_pos, dir)
+				if owner ~= "" and minetest.is_protected(place_pos, owner) then
+					return
```

Comment:
Shouldn't `minetest.record_protection_violation()` be added here?
```suggestion
					minetest.record_protection_violation()
					return true
```

Reply:
This is pointless to call when an automated setup attempts to perform such map actions. The owner might not be online, and protection mods cannot do anything there.